web/src/api/hfiCalculatorAPI.ts

```diff
@@ -7,6 +7,47 @@ import {
 import { DateTime } from 'luxon'
 import { formatISODateInPST } from 'utils/date'
 
+export interface FuelType {
+  abbrev: string
```

Comment:
Needs an `id` field too, yeah?

Reply:
It will do in the next PR - I've added it in now anyway